INTRO.md

```diff
@@ -3,23 +3,22 @@
 ![rangle.io](/img/intro.png)
 
 ## Who is this material for?
-This material is for those who are familiar with ReactJS and are willing to dive into developing mobile apps with [React Native](https://facebook.github.io/react-native). For this course we assume that our audience also has solid knowledge of JavaScript, ES6 syntax, [Redux](https://github.com/reactjs/redux) and CSS.
+This material is for those who are familiar with ReactJS and are willing to dive into developing mobile apps with [React Native](https://facebook.github.io/react-native). For this course we assume that our audience has solid knowledge of JavaScript, ES6 syntax, [Redux](https://github.com/reactjs/redux) and CSS (and Flexbox for layouts).
 
 
 ## What is React Native?
 React Native is a framework for building mobile applications with
 JavaScript and ReactJS by leveraging native UI components.
 
 In ReactJS we have a virtual DOM which reflects the real DOM.
-Each element corresponds to a node in the Virtual DOM and when an element changes, that change is being reflected onto the real DOM. In React Native we don't deal with the DOM but with the Native Components which are provided by specific platforms. Instead of dealing with WebViews we use actual platform specific native components. For example, instead of using HTML elements such as `<div>` & `<span>` we use the native components such as `<View>` & `<Text>`.
+Each element corresponds to a node in the Virtual DOM and when an element changes, that change is being reflected onto the real DOM. In React Native we are not using the DOM but Native Components which are provided by specific platforms. Instead of dealing with WebViews we use actual platform specific native components. 
```

Comment:
> that change is ~~being~~ reflected onto the real DOM

Suggestion: Don't use the present progressive passive tense here.

> Instead of dealing with WebViews, we use actual platform specific native components. 

Suggestion: Add comma.
Aside: This sentence seems a duplicate of the one preceeding it.